client/blocks/jitm/templates/modal-style.scss

```diff
@@ -186,7 +191,6 @@ $wpcom-modal-footer-height: 30px + ( $wpcom-modal-footer-padding-v * 2 );
 
 	font-size: $font-title-medium;
 	line-height: 1;
-	margin: 0 5% 0 0;
```

Comment:
Rely on `.modal__container` to set consistent left and right padding for both the text and heading.